server/datastore/mysql/apple_mdm.go

```diff
@@ -385,20 +385,40 @@ func (ds *Datastore) IngestMDMAppleDevicesFromDEPSync(ctx context.Context, devic
 		return 0, ctxerr.Wrap(ctx, err, "ingest mdm apple host get app config")
 	}
 
+	args := []interface{}{nil}
+	if name := appCfg.MDM.AppleBMDefaultTeam; name != "" {
```

Comment:
Should we have a `nil` check for `appCfg`?

Reply:
I don't think it's necessary, check the implementation of `ds.AppConfig`, besides we're not doing the check anywhere it would fail way earlier!